uniqueidentifier_null.go

```diff
@@ -0,0 +1,49 @@
+package mssql
+
+import (
+	"database/sql/driver"
+)
+
+type NullUniqueIdentifier struct {
+	UUID  UniqueIdentifier
+	Valid bool // Valid is true if UUID is not NULL
+}
+
+func (n *NullUniqueIdentifier) Scan(v interface{}) error {
+	if v == nil {
+		*n = NullUniqueIdentifier{
+			UUID:  [16]byte{0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0},
+			Valid: false,
+		}
+		return nil
+	}
+	u := n.UUID
+	err := u.Scan(v)
+	*n = NullUniqueIdentifier{
+		UUID:  u,
+		Valid: true,
+	}
+	return err
+}
+
+func (n NullUniqueIdentifier) Value() (driver.Value, error) {
+	return n.UUID.Value()
+}
+
+func (n NullUniqueIdentifier) String() string {
+	return n.UUID.String()
+}
+
+func (n NullUniqueIdentifier) MarshalText() (text []byte, err error) {
+	return n.UUID.MarshalText()
+}
+
+func (n *NullUniqueIdentifier) UnmarshalJSON(b []byte) error {
```

Comment:
>b []byte [](http://example.com/codeflow?start=45&length=8)

i think json would use a literal `"null"` string

Reply:
45eb78e835c0e48a881b2f370ea5d6f0f05d9db8 should handle that case